src/main/scala/io/sdkman/changelogs/Migrations.scala

```diff
@@ -35,16 +35,32 @@ class Migrations {
   @ChangeSet(order = "005", id = "005-change_java_default_8u152_zulu", author = "MaSven")
   def migrate005(implicit db: MongoDatabase) = updateCandidateDefault("java", "8u152-zulu")
 
-
   @ChangeSet(order = "006", id = "003-add_maven_353", author = "hho")
   def migrate006(implicit db: MongoDatabase) = {
     insertVersions(CandidateVersion("maven", "3.5.3", Some("UNIVERSAL"), "https://archive.apache.org/dist/maven/maven-3/3.5.3/binaries/apache-maven-3.5.3-bin.zip"))
     updateCandidateDefault("maven", "3.5.3")
   }
 
+  @ChangeSet(order = "008", id = "008-change_openjdk_to_adoptopenjdk", author = "MaSven")
+  def migrate008(implicit db: MongoDatabase) = {
+    updateVersion("9u181-openjdk", "9u181-adoptopenjdk")
```

Comment:
This name is too verbose, and takes up too much real estate in the list view. I think we should stick with openjdk for now.